homeassistant/components/knx/__init__.py

```diff
@@ -146,15 +146,15 @@
 SERVICE_KNX_SEND_SCHEMA = vol.Any(
     vol.Schema(
         {
-            vol.Required(SERVICE_KNX_ATTR_ADDRESS): cv.string,
+            vol.Required(CONF_ADDRESS): cv.string,
```

Comment:
I'd make an internal constant named not `CONF_*` that references `CONF_ADDRESS` so we don't need to use a `CONF_` constant in a service schema which is a bit weird. Also put a comment above that constant why we use it. Ie why we need to make sure that service and config data keys match.